commands.js

```diff
@@ -71,6 +71,10 @@ function Commander (view, cabal) {
           self.view.writeLine.bind(self.view)(`/${key}`)
           self.view.writeLine.bind(self.view)(`  ${self.commands[key].help()}`)
         }
+        self.view.writeLine.bind(self.view)(`ctrl+w`)
```

Comment:
`C-w` is already the shortcut for deleting a word backwards

Reply:
maybe `M-n` and `M-p`?

Reply:
sorry, that's emacs shorthand for ALT+n and ALT+p

Reply:
I couldnt figure out how to ask more complicated shortcuts out of neat. What's the most natural/conventional shortcut for left/right navigation or pane switching on a cli?